src/localization/zh.js

```diff
@@ -1,61 +1,61 @@
 export default {
 
-  version:  `0.1`,
+  version: '2',
```

Comment:
We're not using file-level version numbers any more, so could you update this pull request and remove this line from the google doc?

Reply:
Nevermind, you beat me to it : )